src/calculation/ClauseResultsBuilder.ts

```diff
@@ -589,12 +589,15 @@ export function doesResultPass(result: any | null): boolean {
  * together so that a patient level population relevance set can be made.
  *
  * @private
+ * @param {fhir4.MeasureGroup} populationGroup population group from measure
  * @param {EpisodeResults[]} episodeResultsSet - Results for each episode
+ * @param {string | null} measureScoringCode scoring code for measure (used if scoring code not provided at the group level)
```

Comment:
Yeah I've encountered stuff like this before. It's because the helper function itself returns `string | null`. (undefined is not assignable to type null).

I'm not opposed to making the helper functions return `string | undefined` instead if it makes these function signatures a little easier to understand.

Reply:
Cool I'll try reworking the helper function to return `string | undefined` which should simplify a lot of things